src/test/java/org/sagebionetworks/bridge/services/MasterSchedulerServiceTest.java

```diff
@@ -0,0 +1,202 @@
+        List<MasterSchedulerConfig> mockConfigs = ImmutableList.of(MasterSchedulerConfig.create(), MasterSchedulerConfig.create());
+        when(configDao.getAllSchedulerConfig()).thenReturn(mockConfigs);
+        
+        List<MasterSchedulerConfig> configs = schedulerService.getAllSchedulerConfigs();
+        assertEquals(configs.size(), 2);
+        
+        verify(configDao).getAllSchedulerConfig();
+    }
+    
+    @Test
+    public void testCreateSchedulerConfig() {
+        ObjectNode objNode = JsonNodeFactory.instance.objectNode();
+        objNode.put("a", true);
+        objNode.put("b", "string");
+        
+        MasterSchedulerConfig mockConfig = TestUtils.getMasterSchedulerConfig();
+        
+        when(configDao.createSchedulerConfig(configCaptor.capture())).thenReturn(mockConfig);
+        
+        schedulerService.createSchedulerConfig(mockConfig);
+        MasterSchedulerConfig config = configCaptor.getValue();
+        
+        assertEquals(config.getScheduleId(), SCHEDULE_ID);
+        assertEquals(config.getCronSchedule(), CRON_SCHEDULE);
+        assertTrue(config.getRequestTemplate().get("a").booleanValue());
+        assertEquals(config.getRequestTemplate().get("b").textValue(), "string");
+        assertEquals(config.getSqsQueueUrl(), SQS_QUEUE_URL);
+        assertEquals(config.getVersion(), new Long(VERSION));
+        assertSame(mockConfig, config);
+        
+        verify(configDao).createSchedulerConfig(configCaptor.capture());
+    }
+    
+    @Test(expectedExceptions = InvalidEntityException.class)
+    public void testCreateSchedulerConfigValidates() {
+        MasterSchedulerConfig mockConfig = TestUtils.getMasterSchedulerConfig();
+        mockConfig.setScheduleId(null);
+        
+        schedulerService.createSchedulerConfig(mockConfig);
+        verify(configDao, never()).createSchedulerConfig(any());
+    }
+    
+    @Test
+    public void testGetSchedulerConfig() {
+        MasterSchedulerConfig mockConfig = TestUtils.getMasterSchedulerConfig();
+        
+        when(configDao.getSchedulerConfig(SCHEDULE_ID)).thenReturn(mockConfig);
+        
+        MasterSchedulerConfig config = schedulerService.getSchedulerConfig(SCHEDULE_ID);
+        
+        assertEquals(config.getScheduleId(), SCHEDULE_ID);
+        assertEquals(config.getCronSchedule(), CRON_SCHEDULE);
+        assertTrue(config.getRequestTemplate().get("a").booleanValue());
+        assertEquals(config.getRequestTemplate().get("b").textValue(), "string");
+        assertEquals(config.getSqsQueueUrl(), SQS_QUEUE_URL);
+        assertEquals(config.getVersion(), new Long(VERSION));
+        assertSame(mockConfig, config);
+        verify(configDao).getSchedulerConfig(SCHEDULE_ID);
+    }
+    
+    @Test(expectedExceptions = EntityNotFoundException.class)
+    public void testGetSchedulerConfigThatDoesNotExist() {
+        when(configDao.getSchedulerConfig(SCHEDULE_ID)).thenReturn(any());
```

Comment:
Minor: any() is for matching arguments. If you want to return null, thenReturn(null) will do.